packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -784,19 +784,19 @@ -(NSDictionary *)getUpdatesConfig
     runtimeVersion = _updatesInterface.runtimeVersion ?: @"";
   }
 
-  // url structure for EASUpdates: `http://u.expo.dev/{appId}`
-  // this url field is added to app.json.updates when running `eas update:configure`
+  // the project url field is added to app.json.updates when running `eas update:configure`
   // the `u.expo.dev` determines that it is the modern manifest protocol
   NSString *projectUrl = @"";
   if (_updatesInterface) {
-    projectUrl = [_updatesInterface.updateURL absoluteString] ?: @"";
+    projectUrl = [constants valueForKeyPath:@"manifest.updates.url"];
   }
```

Comment:
I think this can still be from `_updatesInterface.updateURL`? This projectUrl field in the updates config is just the URL that expo-updates is configured to talk to, correct?